service/controller/resource/terminateunhealthynode/metrics.go

```diff
@@ -0,0 +1,32 @@
+package terminateunhealthynode
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	gaugeValue float64 = 1
+)
+
+var (
+	nodeAutoRepairTermination = prometheus.NewGaugeVec(
```

Comment:
How is gauge intended to be used for metrics here? Wouldn't counter make more sense? With `rate()` you'd still get the changes similar to gauge which is mostly `0` and occasionally `1`?

Reply:
the metrics will be 1 or missing or in some super rare cases more than 1 ( if node with same hostname/ip will  ce created and terminated)

Thats why I choose gauge

Reply:
Its emited when node is terminated with label  that has  node name

Reply:
> Its emited when node is terminated with label that has node name

Ah, that makes sense. :+1: